nextflow.config

```diff
@@ -33,11 +44,15 @@ params {
   config_profile_description = false
   config_profile_contact = false
   config_profile_url = false
+
+  seqCenter = false
+  saveReference = false
+  star_memory = false // Cluster specific param required for hebbe
 }
 
 // Container slug. Stable releases should specify release tag!
 // Developmental code should specify :dev
-process.container = 'nfcore/scrnaseq:dev'
+process.container = 'czbiohub/scrnaseq:dev'
```

Comment:
```suggestion
process.container = 'nfcore/scrnaseq:dev'
```

Reply:
I created it - it should be fine now using the `dev` for now and `master` images will be automatically build once we have something more up2date on `master`.